docs/samples/example-dataspace/plato-values.yaml

```diff
@@ -54,9 +54,6 @@ dataplane:
     pullPolicy: Never
     tag: "latest"
     repository: "edc-dataplane-hashicorp-vault"
-  securityContext:
-    # avoids some errors in the log: cannot write temp files of large multipart requests when R/O
-    readOnlyRootFilesystem: false
```

Comment:
can we please leave that in place? otherwise Jersey will vomit exceptions all over the log, because it tries to write a temp file

Reply:
That's the fix of temporary volume. It will fix also the Jersey exceptions :)